src/main/java/com/jfrog/Utils.java

```diff
@@ -9,48 +15,53 @@
  * @author yahavi
  **/
 public class Utils {
-
-    /**
-     * Create the output JSON.
-     *
-     * @param dependencyTree - The dependency tree
-     * @return output JSON.
-     */
-    public static String toJsonString(GradleDependencyTree dependencyTree) {
-        return toJsonString(dependencyTree, "");
+    public static void saveToFileAsJson(File outputFile, GradleDepTreeResults results) {
+        try (FileWriter writer = new FileWriter(outputFile)) {
```

Comment:
Let's use buffering to make sure we don't access the fs for every character.
```suggestion
        try (FileWriter fileWriter = new FileWriter(outputFile);
             Writer writer = new BufferedWriter(fileWriter)) {
```